packages/web/src/components/withCellHOC.tsx

```diff
@@ -83,21 +103,9 @@ export const withCell = ({
   Empty,
   Success,
 }: WithCellProps) => {
-  const isDataNull = (data: DataObject) => {
-    return dataField(data) === null
-  }
-
-  const isDataEmptyArray = (data: DataObject) => {
-    const field = dataField(data)
-    return Array.isArray(field) && field.length === 0
-  }
-
-  const dataField = (data: DataObject) => {
-    return data[Object.keys(data)[0]]
-  }
-
-  const isEmpty = (data: DataObject) => {
-    return isDataNull(data) || isDataEmptyArray(data)
+  // If its prerendering, render the Cell's Loading component
+  if (isPrerendering()) {
```

Comment:
Maybe we can just have this as a boolean? Does it have to be determined at runtime?

Reply:
I've done this to intentionally hide way the implementation, incase we want to change it later.